### DISCUSSION TITLE
🖼️ Any need for icon shortcuts?

### DISCUSSION THREAD

**Discussion by OCram85** [thuliteio/doks#274] [Ideas]
If needed, I would like to contribute shortcuts for icon libraries like

- [Feather](https://feathericons.com/)
- [FontAwesome](https://fontawesome.com/)

Both would be implemented in this style:

- `{{< Feather icon="music" >}}`
- `{{< FontAwesome icon="fas fa-music" >}}`

> :bulb: Both shortcuts could have optional parameters

**Comment by vidokun** [comment]
Wow, great idea ..

**Reply by epogrebnyak** [reply]
Could this be even shorter? 

```
{{< FontAwesome "fas fa-music" >}}
```

**Reply by OCram85** [reply]
I would prefer shortcuts with named parameters. Just in case there could be multiple params...

**Comment by epogrebnyak** [comment]
Makes sense!

пн, 28 июн. 2021 г., 16:29 Marco Blessing ***@***.***>:

**Comment by bbardsley** [comment]
Hi, how did you import fontawesome anyway? I don't seem to be succesful at it.

**Comment by OCram85** [comment]
I didn't have much time working on this.

I guess I going to try this way:
 -  Copy webfonts and css files in the `static` folder.
 - Search for the scss files where they need to be imported
 - Maybe there is some js or postcss magic 🧙 
 
Best case would be:
- The required font packs should only be imported while rendering, if they were used.
  - Otherwise we would deploy all the resources even if they are not used.
